UefiDbgExt/dbgexts.cpp

```diff
@@ -0,0 +1,351 @@
+/*++
+
+Copyright (c) 2000  Microsoft Corporation
```

Comment:
I assume some of these filers are old boilerplate but the copyright headers are mixed and with various flavors of Microsoft copyright and some are missing `SPDX-License-Identifier: BSD-2-Clause-Patent`.

I suggest we clean them up and make them consistent within this code tree.